doc/1.4/language.md

```diff
@@ -2823,6 +2823,20 @@ where each input parameter is of [*serializable type*](#saved-variables).
 This means that `after` statements cannot be used with methods
 that e.g. have pointer parameters.
 
+Any events posted via an `after` statement are associated with the object
+that the method using the statement belongs to. It is possible to cancel or
+poll all `after` events associated with an object through that object's
+`remove_after()` or `posted_after()` method, as provided by the
+[`after_domain` template](dml-builtins.html#after_domain).
+
+<div class="note">
+
+**Note:** We plan to extend the `after` statement to allow for users to
+explicitly state what objects that events posted from the use of that statement
+are to be associated with.
```

Comment:
The formulation here is a bit awkward. Does anyone have a better suggestion?

Reply:
Empty string? We have no obligation to advertise not-yet-implemented features. We probably won't add this until someone asks for it.

Reply:
Don't you remember the debate? Leonid has already asked for it. The only reason we decided the current design was acceptable was _because_ we would support explicitly associating objects with after statements.

Reply:
"explicitly state which object to associate with the event"?

You can ponder if this is best written here or in the release note.

Reply:
I think it's best written here. I've tinkered the reformulation a bit -- I want to say "associate events with objects" rather than "associate objects with events", as the objects are the "owners" of the events, in some sense.